suricata/update/main.py

```diff
@@ -619,9 +619,11 @@ def dump_sample_configs():
 def resolve_flowbits(rulemap, disabled_rules):
     flowbit_resolver = rule_mod.FlowbitResolver()
     flowbit_enabled = set()
+    pass_ = 1
     while True:
+        logger.debug("Checking flowbits for pass %d of rules.", pass_)
         flowbits = flowbit_resolver.get_required_flowbits(rulemap)
-        logger.debug("Found %d required flowbits.", len(flowbits))
+        logger.debug("Found %d required flowbits (pass %d).", len(flowbits), pass_)
         required_rules = flowbit_resolver.get_required_rules(rulemap, flowbits)
         logger.debug(
             "Found %d rules to enable to for flowbit requirements",
```

Comment:
Update this line to include the pass number... For example:
```
Found %d rules to enable to for flowbit requirements (pass %d)
```

Gives us a little cleaner output.

Reply:
Just a tiny doubt about the semantics of the log here - would this be better worded as :
```
Found %d rules to enable for flowbit requirements (pass %d)
```
(I just removed the second 'to')

Reply:
Yes, that is better.  Thanks.